android/src/main/res/layout/activity_main.xml

```diff
@@ -16,6 +16,39 @@
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent">
+
+        <SearchView
```

Comment:
This should be a direct child of the `ConstraintLayout` instead of the `MapView`.
e.g.
```
<ConstraintLayout>
  <MapView/>
  <SearchView/>
  <RecyclerView/>
</ConstraintLayout>
```

You can then use constraints to make each widget positioned/overlap properly.